### PR TITLE
Add info bounty proposals

### PR DESCRIPTION
**Pull request by filippoweb3**
The curator assignment proposal must be executed after the bounty has been accepted and funded.

### CODE REVIEW

File: docs/learn/learn-guides-bounties.md [hunk 1]
```diff
@@ -56,6 +56,19 @@ Submitting a bounty proposal will reserve
 
 ## Assign a Curator to a Bounty
 
+:::info
+
+The curator assignment call can only be executed successfully if the original bounty proposal
+funding has been executed and a new spending period has not begun. Proposers need to ensure they
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]: "has begun" or "has not begun"? The followup statement seems to suggest otherwise.

File: docs/learn/learn-guides-bounties.md [hunk 2]
```diff
@@ -56,6 +56,19 @@ Submitting a bounty proposal will reserve
...
+calculate a time further past the new spending period for the execution of their curator assignment
+call. Suppose your bounty has been awarded and gets funded at the end of the current spending
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]: Aren't all approved bounties funded at the end of the spending period during which they got approved?

File: docs/learn/learn-guides-bounties.md [hunk 3]
```diff
@@ -56,6 +56,19 @@ Submitting a bounty proposal will reserve
...
+period. In that case, you need to make sure that if the curator assignment ends before the end of
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]: "Curator assignment ends"  - do you mean "Curator assignment referenda" is approved before the end of the spend period and the enactment of the call needs to happen after?

**Review by DrW3RK — COMMENTED** [review]
As a followup PR, I think it makes sense to add a guide with instructions in text with screenshots and place this info banner right before the curator call screenshot. Without the context, it was hard to follow what info is being conveyed.

**Comment by filippoweb3** [comment]
@DrW3RK feedback addressed. I also got additional feedback from Anaelle. I moved the banner at the and of the section. The idea is that people watch the video to understand the flow. Doing a written tutorial with screenshots is redundant (and heavy) as Paritly already has those and I created the tutorials using them to ease the learning process

**Comment by DrW3RK** [comment]
> Paritly already has those

Could you link them to the Wiki? 

The Wiki is text first, and screenshots/videos are augmented resources. Textual instructions are always available on the Wiki or will always be linked to on external pages like support docs.

File: docs/learn/learn-guides-bounties.md [hunk 4]
```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
 [Bounties page](https://polkadot.js.org/apps/#/bounties). Only after the curators claim their
 candidacy the bounty will show as "active" on the main Bounty page.
 
+:::info Curator assignment must execute after bounty funding
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]:
```suggestion
:::info Curator assignment call must be executed after bounty is funded
```

File: docs/learn/learn-guides-bounties.md [hunk 5]
```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
...
+
+Curator assignment must take place after the bounty has been funded, and approved bounties can only
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]: "can only be funded"  or "are only funded"?

Also, at the beginning of next paragraph, you mention that they are funded (through the treasury) at the end of current spending period, where as here you mention they "can be funded after a spend period has been completed".

We may have to rephrase that

File: docs/learn/learn-guides-bounties.md [hunk 6]
```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
...
+be funded after a spend period has been completed. A curator assignment referendum cannot be
+executed during the same spend period as the bounty funding.
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]:
```suggestion
executed during the same spend period when the bounty is funded.
```

File: docs/learn/learn-guides-bounties.md [hunk 7]
```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
...
+
+If your bounty has been awarded, it will be funded at the end of the current spending period. You
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]: This first sentence will be unnecessary, if it is clarified already before.

File: docs/learn/learn-guides-bounties.md [hunk 8]
```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
...
+need to make sure that if the curator assignment referendum ends before the end of the spending
+period, the enactment of the curator assignment referendum will take place in the next spending
+period (you can set a delay in
+[Polkadot-JS](./learn-guides-polkadot-opengov.md#submitting-a-proposal) when you submit the
+proposal).
```
Comment (profradha) [inline docs/learn/learn-guides-bounties.md]: rephrased a bit. 

```suggestion
need to make sure that the curator assignment referendum gets enacted in the subsequent spending
period (you can set a delay in
[Polkadot-JS](./learn-guides-polkadot-opengov.md#submitting-a-proposal) when you submit the
proposal during the current spend period).

```

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: docs/learn/learn-guides-bounties.md (hunk 5 above)
Comment (filippoweb3) [reply docs/learn/learn-guides-bounties.md]: changed

File: docs/learn/learn-guides-bounties.md (hunk 7 above)
Comment (filippoweb3) [reply docs/learn/learn-guides-bounties.md]: deleted

**Comment by filippoweb3** [comment]
@DrW3RK Anaelle told me we can add the links. I added them as banners at the start of each section.

**Review by DrW3RK — APPROVED** [review]
